runtime/src/launcher/kotlin/konan/start.kt

```diff
@@ -21,9 +21,7 @@ private fun Konan_start(args: Array<String>): Int {
         return 0
 
     } catch (e: Throwable) {
-        // TODO: may be add some more info.
-        print("Uncaught exception from Kotlin's main: ")
-        e.printStackTrace()
+        kotlin.native.internal.OnUnhandledException(e)
```

Comment:
Maybe use `TerminateWithUnhandledException` here?
Then such a termination would be detected as a crash by a debugger. Do we want application to behave like that?